src/routes/mainPage.jsx

```diff
@@ -71,8 +71,9 @@ export default function MainPage() {
   useEffect(() => {
     axios
       .get(url)
-      .then((data) => {
-        setGoodWords(data.name);
+      .then((response) => {
+        setGoodWords(`${response.data.content} / ${response.data.name}`);
+        console.log(response);
```

Comment:
```suggestion
        setGoodWords(`${response.data.content} - ${response.data.name}`);
```
콘솔에 `response`를 보여주는 것은 디버깅 옵션이 있을때 하면 좋겠습니다. 일단 없애는게 어떨까요?